.misc/deploy.pypi.sh

```diff
@@ -11,11 +11,25 @@ echo "Uploading coala to pypi"
 pip3 install twine wheel
 python3 setup.py sdist bdist_wheel
 # Upload one by one to avoid timeout
-# Don't block merging because pypi is in error 500 mood again
-set +e
-twine upload dist/*.whl -u "$PYPIUSER" -p "$PYPIPW"
-twine upload dist/*.tar.gz -u "$PYPIUSER" -p "$PYPIPW"
-set -e
+error=$( twine upload dist/*.whl -u "$PYPIUSER" -p "$PYPIPW" 2>&1 | tee /dev/tty ; echo "status: ${PIPESTATUS[0]}" )
+if [ "${error: -9}" == "status: 0" ]; then
+   echo "Uploaded wheel to pypi"
+else
+    if [[ $error == *"500"* ]]; then
+        echo "Server error 500"
+        exit
+    fi
+fi
+
+error=$( twine upload dist/*.tar.gz -u "$PYPIUSER" -p "$PYPIPW" 2>&1 | tee /dev/tty ; echo "status: ${PIPESTATUS[0]}" )
+if [ "${error: -9}" == "status: 0" ]; then
+   echo "Uploaded tar to pypi"
+else
+    if [[ $error == *"500"* ]]; then
+        echo "Server error 500"
+        exit
+    fi
+fi
```

Comment:
You dont need to create variable error. tee the pipe to a temp file that you can load afterwards.